airflow/sensors/base.py

```diff
@@ -155,8 +171,8 @@ class BaseSensorOperator(BaseOperator, SkipMixin):
         and AirflowFailException, the sensor will log the error and continue
         its execution. Otherwise, the sensor task fails, and it can be retried
         based on the provided `retries` parameter.
-    :param never_fail: If true, and poke method raises an exception, sensor will be skipped.
-           Mutually exclusive with soft_fail.
+        Mutually exclusive with soft_fail.
+    :param skip_policy:
```

Comment:
Description needs to be added, probably with details about each option in the enum.
I'd include answers to the following questions:
1. What is a "soft" error?
2. What's the difference between "ignore" and "skip"? (it might not be as trivial for the user)